packages/plugins/response-cache/src/index.ts

```diff
@@ -34,9 +39,27 @@ export function useResponseCache(options: UseResponseCacheParameter): Plugin {
           ...options,
           cache,
           session: sessionFactoryForEnvelop,
+          buildResponseCacheKey: cacheKeyFactoryForEnvelop,
         }),
       )
     },
+    async onRequest({ request, fetchAPI, endResponse }) {
+      if (enabled(request)) {
+        const operationId = request.headers.get('If-None-Match')
+        if (operationId) {
+          const cachedResponse = await cache.get(operationId)
```

Comment:
Maybe we should also support `has` on the cache (and use get as a fallback). It can be faster in the case of external storage such as Redis (https://redis.io/commands/exists/). 🤔

Reply:
We need the data for `timestamp`. Maybe we can improve the logic for the timestamp later.